components/brave_wallet/browser/internal/hd_key_sr25519.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_SR25519_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_SR25519_H_
+
+#include <memory>
+
+#include "base/containers/span.h"
+#include "brave/components/brave_wallet/browser/internal/polkadot/rust/sr25519.h"
+
+namespace brave_wallet {
+
+namespace polkadot {
+class SchnorrkelKeyPair;
+}  // namespace polkadot
+
+using polkadot::SchnorrkelKeyPair;
+using polkadot::SR25519PublicKey;
+using polkadot::SR25519Signature;
+
```

Comment:
`usign smth` is not ok in headers in namesapce scope
https://google.github.io/styleguide/cppguide.html#Namespaces